spec/features/users_creates_new_account_spec.rb

```diff
@@ -1,15 +1,15 @@
 require "rails_helper"
 
 feature "user creates new account" do
-  scenario "successfully" do
+  scenario "successfully without login" do
```

Comment:
I think the description was descriptive enough without the addition but really up to you